provider/import.go

```diff
@@ -130,6 +134,16 @@ func Import(ctx context.Context, p Provider, hcl, tfstate writer.Writer, f *filt
 						return errors.Wrapf(err, "error while calculating the satate of resource %q", t)
 					}
 				}
+				// we construct a map[string]string to perform
+				// interpolation later. By default, the value to interpolate is the ID
+				// but in some case (ex: google), it can be different.
```

Comment:
Different how?

Reply:
actually the comment is misleading. I modified it consequently. 
> it can be different

resource key could be `self_link` with Google.